lib/harmony/harmonydb/sql/20230712.sql

```diff
@@ -0,0 +1,37 @@
+create table SectorLocation
+(
+    "miner_id"        int8,
+    "sector_num"   int8,
```

Comment:
Why int8, and not just int (/maybe bigint which is more standard)?

Reply:
bigint is the same as int8. I created the columns with bigint and the ddl generated showed int8